app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional it appears blank on start up. 
+type: troubleshooting
+page_title: Upgrading from Kendo UI Core to Professional Breaks App
+slug: app-broken-after-kendo-upgrade
+position: 0
+tags: kendo, package manager, blank, references, undefined
+ticketid: 1017633
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When the Kendo UI package type is changed from Kendo UI Core to Kendo UI Professional (and vice versa) in an app using the Package Manager, all styles and functionality may be lost.
+
+## Cause
+When the Kendo UI package type is changed, the script and style files are replaced with new ones that have different names. This breaks the script and style references in the app.
+
+## Solution
+You need to update the Kendo UI references in your app's *index.html* file. Replace the files names in the references as follows:
```

Comment:
* in your `index.html`
* the files names -> the file names

We're usually going with code formatting for file names.